src/lib/tuning/index/index_evaluator.cpp

```diff
@@ -0,0 +1,75 @@
+#include "index_evaluator.hpp"
+
+#include "optimizer/column_statistics.hpp"
+#include "optimizer/table_statistics.hpp"
+#include "resolve_type.hpp"
+#include "storage/index/base_index.hpp"
+#include "storage/storage_manager.hpp"
+#include "storage/table.hpp"
+#include "types.hpp"
+#include "utils/logging.hpp"
+
+namespace opossum {
+
+IndexEvaluator::IndexEvaluator() {}
+
+void IndexEvaluator::_setup() { _saved_work.clear(); }
+
+void IndexEvaluator::_process_access_record(const BaseIndexEvaluator::AccessRecord& record) {
+  auto table_statistics = StorageManager::get().get_table(record.column_ref.table_name)->table_statistics();
+  // ToDo(anyone) adapt for multi column indices...
```

Comment:
I believe we should add that to the wiki entry.

Reply:
You could add this paragraph to the end of our wiki entry:
> As there is currently no operator benefiting from the use of multi-column indexes (multi-column scans are implemented by a sequence of multiple single-column scans), they are not considered during tuning, but nevertheless supported by the underlying structures.